src/utils/constants.ts

```diff
@@ -1,5 +1,7 @@
 import { Option } from "./types";
 
+export const OCL_URL = "https://app.openconceptlab.org/";
```

Comment:
So, my suggestion here was meant to work more like the constants below, that is:

```ts
export const OCL_URL: string =
  // @ts-ignore OCL_URL is injected at runtime
  window.OCL_URL ||
  "https://app.staging.openconceptlab.org/#/";
```

Then we can adjust `OCL_SIGNUP_URL` to be defined this way:

```ts
export const OCL_SIGNUP_URL = OCL_URL + "accounts/signup";
```

The properties defined here are injected via environment variables so that we can drive locations by configuration rather than by hard-coding like `getEnv()` does.